test/test_helper.rb

```diff
@@ -68,6 +60,29 @@ def stub_module(name)
   end
 end
 
+# stub missing YaST modules from different yast packages to avoid build dependencies
+
+# these are not used in the tests so we can just use an empty implementation
+Yast::RSpec::Helpers.define_yast_module("FTP")
+Yast::RSpec::Helpers.define_yast_module("HTTP")
+Yast::RSpec::Helpers.define_yast_module("NtpClient")
+Yast::RSpec::Helpers.define_yast_module("Proxy")
+
+# define missing modules with an API, these are used in the tests and need to
+# implement the *same* API as the real modules
+
+Yast::RSpec::Helpers.define_yast_module("InstFunctions", methods: [:second_stage_required?])
+
+Yast::RSpec::Helpers.define_yast_module("Language", methods: [:language]) do
+  # @param _lang [String]
+  # @return [Boolean]
+  def supported_language?(_lang); end
+
+  # @param _force [Boolean]
+  # @return [Hash<String,Array>]
+  def GetLanguagesMap(_force); end
```

Comment:
with change how method is defined, this won't be needed.

Reply:
no, see https://github.com/yast/yast-ruby-bindings/pull/281#discussion_r786956702